bagger/README.md

```diff
@@ -0,0 +1,229 @@
+configuration to authenticate to Wasabi (see below for details on how these variables are used in
+DART). However, only ReBACH-Bagger uses the other variables defined here to access the endpoint for
+the purpose of checking for duplicate bags. DART uses the storage configuration embedded in the
+workflow JSON file for selecting the correct endpoint. Verify that these values match.
+
+```toml
+[Wasabi]
+name = "***override***"
+host = "***override***"
+bucket = "***override***"
+host_bucket = "***override***"
+access_key = "***override***"
+secret_key = "***override***"
+```
+
+## Metadata Configuration
+
+ReBACH-Bagger can apply a configurable set of metadata tags to the bags it generates. These tags
+are defined in the `Metadata` section of the configuration file. The tags are defined with the
+following schema:
+
+`# tag-file.Tag-Name = "Tag value"`
+
+The `tag-file` element corresponds to the tag file (e.g. bag-info.txt) in which the tag will be
+placed. Do not include `.txt` as part of the tag-file element.
+
+`Tag-Name` is the name of the metadata tag. Conventionally, tag names are uppercase, dash-separated
+words.
+
+If the `"Tag value"` element is a string, ReBACH-Bagger will simply use the string as the value of
+the tag.
+
+### Metadata from JSON
+
+Users may also use an inline table to define a dot-notation `tag_path` to the tag's corresponding 
+key in the package's metadata file. Take the following abbreviated example of a metadata JSON file:
+
+```json
+{
+  "authors": [
+    {
+      "full_name": "Brian Avants"
+    },
+    {
+      "full_name": "Arno Klein"
+    }
+  ],
+  "license": {
+    "name": "Apache 2.0",
+    "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+  },
+  "title": "MNI space average T1-weighted MRI of the brain with anatomical labels.",
+  "doi": "10.6084/m9.figshare.2066037.v17"
+}
+```
+
+To define a set of tags based on this metadata named "First-Author", "License", and "DOI" in the 
+"bag-info.txt" tag file, users can define the following relationships in the config file:
+
+```toml
+[Metadata]
+bag-info.First-Author = { tag_path = "authors.0.full_name" }
+bag-info.License = { tag_path = "license.name" }
+bag-info.DOI = { tag_path = "doi" }
+```
+
+### Metadata Utilities
+
+ReBACH-Bagger can strip HTML tags out of a metadata value. To enable this functionality, use a
+TOML inline table to define a `tag_path` key and set the `strip_html` key to `true`. Optionally,
+set the `shorten` key to the maximum number of characters to allow in the formatted value.
+Shortening will truncate at word boundaries (see
+[Python documentation](https://docs.python.org/3.9/library/textwrap.html#textwrap.shorten)), so
+the resulting string may be shorter than the specified limit.
+
+```toml
+[Metadata]
+aptrust-info.Description = { tag_path = "description", strip_html = true, shorten = 100 }
+```
+
+## DART Workflow
+
+A [DART workflow](https://aptrust.github.io/dart-docs/users/workflows/) is a JSON file that
```

Comment:
A [DART workflow](https://aptrust.github.io/dart-docs/users/workflows/) produces a JSON file

Reply:
I've changed it to read "can be represented as a JSON file"